efcms-service/serverless.yml

```diff
@@ -381,6 +381,26 @@ resources:
             MetricsEnabled: true
 
 functions:
+  createDocument:
+    handler: src/documents/createDocumentLambda.createDocument
+    events:
```

Comment:
When `docker-run.sh` finishes running, it doesn't report that the application is ready. It just reports, e.g.:

> ✨  Built in 43.58s.

Please provide a status message reporting that the build process is finished and the site can be accessed, preferably specifying the URL (i.e., `http://localhost:1234`) where it can be used.